packages/odyssey-storybook/src/components/odyssey-mui/Toast/Toast.stories.tsx

```diff
@@ -32,11 +34,19 @@ export default {
     },
   },
   argTypes: {
+    action: {
+      control: "text",
+      default: null,
+    },
     content: {
       control: "text",
       defaultValue:
         "The mission to Sagitarius A has been scheduled for January 7.",
     },
+    isDismissible: {
+      control: "boolean",
+      defaultValue: null,
+    },
```

Comment:
Are we adding this prop or is it MUI?

I prefer positive booleans, so you don't end up with double-negatives. This one's kinda weird because "dismiss" is negative, but we're talking about enabling, not disabling functionality.

Reply:
This will be a custom prop specific to Toast. MUI's Alert handles this by detecting whether or not `onClose` and `action` have values, but we will want to set those ourselves under the hood to ensure consistency (rather than expose them to Odyssey users).

Our default behavior is for Toasts to auto-dismiss rather than be manually dismissible, hence the name for this flag. I'm happy to revisit the props once we wrap and export because we may want to change the associated JS as well.